docs/content/utilities/moose_docs/setup.md

```diff
@@ -75,15 +80,15 @@ python configuration you have will not be damaged.
 conda create --name docs --clone root
 ```
 
-(2) Activate the new environment.
+(3) Activate the new environment.
 
 This command will need to be executed within any terminal that you plan to perform documentation related tasks.
 
 ```bash
 source activate docs
 ```
 
-(3) Install the additional python packages.
+(4) Install the additional python packages.
 
 ```bash
 pip install python-markdown-math bs4 pybtex jinja2 livereload lxml pylatexenc anytree
```

Comment:
can you add "pandas" to this please, this is something that I should of done a while ago.